### DISCUSSION TITLE
關於Lab1 有了2個文件之後 如何接下去到jupyter notebook的啓動

### DISCUSSION THREAD

**Discussion by HwangYangHong** [bol-edu/HLS-SOC-Discussions#22] [Q&A]
想請問有了2個文件之後，該如何進到有著相對應的code的jupyter notebook呢？
![Result5 （the two files）](https://user-images.githubusercontent.com/62707214/192282583-d1c4156a-7c5b-4312-9d23-12669cebc7c0.PNG)

**Comment by ciwade870307** [comment]
Please follow the **onlineFPGA user manual** to rent a pynq device board. Then, with the given IP and password, you can access the jupyter web and upload these two bitstream-related files.

If you don't have the onlineFPGA user manual, please **contact your TA** due to the privacy of the manual.
